src/components/FeaturesDemo/CodeEditor/styles.module.scss

```diff
@@ -0,0 +1,90 @@
+.wrapper {
```

Comment:
Why you choose scss instead stylus?

Reply:
Sass seems to be far more popular than stylus. IDK, I'm used to working with Sass and I think they all are mostly the same. Do you have any reasons to choose stylus.

Most importantly, I don't think we need sass much at all, because Tailwind covers almost all our styling needs.